Add onno-vos-dev implementation

onno-vos-dev opened the pull request:
- Add onno-vos-dev submodule
- Make paths in shell scripts passable as cli arg rather than hardcoded to specific path
- Use gtime on MacOS rather than /usr/bin/time
- Split tables into two, one for 50M and one for 1B so they can be sorted by time
- Add back Jesperes run as it got lost when using garazdawi implementation

onno-vos-dev committed:
Add onno-vos-dev implementation

- Add onno-vos-dev submodule
- Make paths in shell scripts passable as cli arg rather than hardcoded to specific path
- Use gtime on MacOS rather than /usr/bin/time
- Split tables into two, one for 50M and one for 1B so they can be sorted by time
- Add back Jesperes run as it got lost when using garazdawi implementation

IceDragon200 commented on scripts/impls/1B/onno-vos-dev.sh:
```diff
@@ -0,0 +1,11 @@
+mkdir -p "${outdir}"
+data_path=$1
+
+# This is not an error, garazdawi's implementation was based off jesperes'
```
You can remove this comment now, this was more so for me

IceDragon200 commented on common.sh:
```diff
@@ -1,5 +1,14 @@
+if [ "$(uname)" == "Darwin" ]; then
+  timecmd="gtime -v"
+elif [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+  timecmd="time -v"
```
This actually defaults to the shell built-in time on linux, that's why I was using `/usr/bin/time` to force GNU time

onno-vos-dev replied:
Ok I can switch to /usr/bin/time but the `-v` option is not available on Mac so I'll leave the `gtime` if-handling in place OK?

IceDragon200 replied:
That's fine, I was only concerned with linux for now (until someone runs in here with a windows version, ugh)

IceDragon200 commented on scripts/impls/50M/onno-vos-dev.sh:
```diff
@@ -0,0 +1,11 @@
+mkdir -p "${outdir}"
+data_path=$1
+
+# This is not an error, garazdawi's implementation was based off jesperes'
```
You can remove the comment

IceDragon200 commented on setup.sh:
```diff
@@ -59,3 +59,8 @@ Brc.main("./measurements.txt", 50_000_000)
+
+#
+cd "${impldir}/onno-vos-dev/1brc"
+rebar3 compile
```
I don't have rebar3 available on my local since I work primarily with elixir, so I wouldn't be able to run your implementation like this, unless you can figure out how to run rebar3 from mix

onno-vos-dev replied:
Let me fix that 👍

onno-vos-dev committed:
Fix review comments

- Remove obsolete comments
- Make code rebar3 independant
- Use /usr/bin/time instead of just time

IceDragon200 commented on common.sh:
```diff
@@ -1,5 +1,14 @@
-#!/usr/bin/env bash
-timecmd="/usr/bin/time -v"
+#!/usr/bin/env bas
```
I think you ate the `h` off bash here D:

onno-vos-dev committed:
Do not eat h characters

IceDragon200 commented:
Okay LGTM, if anything else is missing or wrong I should be able to fix it
